perun-base/src/main/java/cz/metacentrum/perun/core/api/UserExtSource.java

```diff
@@ -19,6 +20,7 @@
 	 * false = UserExtSource can be removed. It is truly external.
 	 * true = UserExtSource can NOT be removed. It is somehow important and needed in the system. */
 	private boolean persistent;
+	private LocalDate lastAccess = LocalDate.now();
```

Comment:
At the end we decided, that ``lastAccess`` param is mostly informative. We will make it ``String`` in our general format, like ``createdAt`` and ``modifiedAt`` are. It will be simpler for API consumer, since parsing of such params is already implemented.